.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+## Status
+**READY/IN DEVELOPMENT/HOLD**
+ - [ ] Ready
+ - [ ] In development
+ - [ ] Hold
+
+## Closes issue(s)
+A list of issues that are closed by this PR
+
+## Description
+A few sentences describing the overall goals of the pull request's commits.
+
+# Changes include
+- [ ] Bugfix (non-breaking change that solves an issue)
+- [ ] New feature (non-breaking change that adds functionality)
+- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)
+- [ ] Refactor (non-breaking/breaking change that refactor the previous code base)
+
```

Comment:
I think we can add this to the Todo and remind people to use labels when creating their PRs.

Reply:
I think its better to keep the todo separate and the changes separate. 
TODO's are basically code code sanitizing steps - I will just name that as checks I thinks.